ui/temboardui/web/tornado.py

```diff
@@ -289,6 +289,8 @@ def instance_middleware(request, address, port, *args):
             # Swallow adddress and port arguments.
             request.instance = cls(request)
             request.instance.fetch_instance(address, port)
+            if not hasattr(request, "json"):
```

Comment:
Un petit commentaire genre:

*# Disable status prefetch for API as we prefetch status for `base.html` template.*